src/phoenix/server/api/helpers/dataset_helpers.py

```diff
@@ -133,3 +137,42 @@ def _get_generic_io_value(
     ):
         return io_value
     return {}
+
+
+def _get_message(message: Mapping[str, Any]) -> Dict[str, Any]:
+    content = get_attribute_value(message, MESSAGE_CONTENT)
```

Comment:
Not blocking this PR, but it might simplify the query to just query for `models.Span` and nix the whole `HasSpanIO` stuff. I did not remember we had this `get_attribute_value` helper function.

Reply:
ok

Reply:
will punt on this